build/targets/PackageVersions.props

```diff
@@ -27,7 +30,7 @@
     <!-- Visual Studio packages -->
```

Comment:
2 lines above this, `MicrosoftVisualFSharpMSBuild150PackageVersion` can go.  There are probably also old references to this package in various `packages.config` files.

Reply:
@brettfo, yeah still working on this :-)